src/mds/Server.cc

```diff
@@ -2695,7 +2695,7 @@ void Server::dispatch_client_request(const MDRequestRef& mdr)
     respond_to_request(mdr, mdr->more()->peer_error);
     return;
   }
-  
+
```

Comment:
please drop this from `qa/cephfs/test_quiesce: test proper handling of remote authpins`